pkg/bridgeclient/app.go

```diff
@@ -32,6 +33,8 @@ type Arguments struct {
 	cloudEvents    *bool
 }
 
+var quitChannel = make(chan os.Signal, 1)
```

Comment:
To collect integration test coverage, we need to be able to exit the apps gracefully, which is what this quitChannel is for. When running apps in `test` mode, we listen for a specific NATS message that then sends a signal using this channel, which is then a trigger to exit the app.